src/pages/home/report/ReportActionItem.js

```diff
@@ -269,6 +272,8 @@ class ReportActionItem extends Component {
             <>
                 <PressableWithSecondaryInteraction
                     ref={el => this.popoverAnchor = el}
+                    onPressIn={() => this.props.isSmallScreenWidth && ControlSelection.block()}
+                    onPressOut={() => ControlSelection.unblock()}
```

Comment:
NAB: Why not include `this.props.isSmallScreenWidth && ` here too?

Reply:
This Change will block text selection at the root. But we have a feature where we select text on the web and then open the context menu to copy the selected part.